src/pages/account/profile.tsx

```diff
@@ -14,20 +15,112 @@ const profile = () => {
       setProfile(data);
     });
   }, []);
-  return (
-    <div className="flex-auto h-screen w-full justify-center place-items-center bg-backgroundColor">
+  const getBirthDate = () => {
+    const months = [
+      "Jan",
+      "Feb",
+      "Mar",
+      "Apr",
+      "May",
+      "Jun",
+      "Jul",
+      "Aug",
+      "Sep",
+      "Oct",
+      "Nov",
+      "Dec",
+    ];
+    const month = profile?.birthdate.toString().split("T")[0].split("-")[1];
+    const month_str = months[parseInt(month!) - 1];
+    const day = profile?.birthdate.toString().split("T")[0].split("-")[2];
+    const year = profile?.birthdate.toString().split("T")[0].split("-")[0];
+    return (
+      <div>
+        Birth:{" "}
+        <strong>
+          {day}/{month_str}/{year}
+        </strong>
+      </div>
+    );
+  };
```

Comment:
Why are you doing this? In `UserProfile` the birthdate is a `Date` object right? Couldn't you just use the built-in method? [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)